src/imu/sensors/imu_sensors/sensor_with_sensitivity_virtual_interface.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef SRC_sensor_with_sensitivity_INTERFACE_HPP_
+#define SRC_sensor_with_sensitivity_INTERFACE_HPP_
+
+#include "imu_sensitivity.hpp"
+#include "mpu9255_data.hpp"
+#include "sensor_vector.hpp"
+
+namespace imu {
+
+class SensorWithSensitivityInterface : public SensorVector {
```

Comment:
Because of the polution of the diagram. They are only needed for unittesting. But I added all of them. Will refactor the architecture later to composition over inheritance. That will clean up the architecture a bit.